src/tests/test_dal.py

```diff
@@ -0,0 +1,32 @@
+from polydatum import DataAccessLayer, DataManager as _BaseDataManager
+
+
+def test_custom_dal():
+    """
+    Verify that a custom DataAccessLayer can be passed to DataManager
+    """
+
+    class TestDal(object):
+        """
+        Test dal class
+        """
+        def __init__(self, data_manager):
+            self._services = {}
+            self._data_manager = data_manager
+
+    class TestDataManager(_BaseDataManager):
```

Comment:
The prefix `Test` should be avoided. py.test will consider class with the the prefix `Test` to be test cases and logs warnings when it's not actually a test. We use `Sample` or `Example` for this reason.